internal/util/argutil/argutil.go

```diff
@@ -85,3 +89,20 @@ func ParseFieldPath(path string) ([]string, error) {
 	}
 	return newParts, nil
 }
+
+// ResolveSymlink returns the resolved symlink path for the input path
+func ResolveSymlink(ctx context.Context, path string) (string, error) {
+	isSymlink := false
+	f, err := os.Lstat(path)
```

Comment:
shouldn't we fail fast here and return on err ?

Reply:
Yes. Updated.